docs/workflows/sign-images.rst

```diff
@@ -298,3 +298,27 @@ It is possible to specify a single manifest identified by tag or a list of manif
 by proviging ``tags_list`` option to the call.
 Note that ``manifest lists`` are not signed, instead all the image manifests that manifest lists
 contain, are signed.
+
+Push signed images to the Pulp Registry
+=======================================
+
+Signatures can be pushed to the Pulp Registry via the extensions API. This API exposes an endpoint
```

Comment:
can you mention that containers registries.d file sigstore section needs to be configured accordingly? or point to the specs https://github.com/containers/image/blob/main/docs/containers-registries.d.5.md

Reply:
on the client side.